Last-Update: 2012-12-13
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Forwarded: no
Description: lintianisation of man pages

--- a/doc/glmark2.1.in
+++ b/doc/glmark2.1.in
@@ -49,7 +49,7 @@
 Size of the output window (default: 800x600)
 .TP
 \fB\-\-fullscreen\fR
-Run in fullscreen mode (equivalent to --size -1x-1)
+Run in fullscreen mode (equivalent to \-\-size \-1x\-1)
 .TP
 \fB\-\-results\fR RESULTS
 The types of results to report for each benchmark, as a ':' separated list [fps,cpu,shader]
@@ -74,7 +74,7 @@
 .TP
 \fB\-\-annotate\fR
 Annotate the benchmarks with on-screen information
-(same as -b :show-fps=true:title=#info#)
+(same as \-b :show-fps=true:title=#info#)
 .TP
 \fB\-d\fR, \fB\-\-debug\fR
 Display debug messages
@@ -114,20 +114,20 @@
  'shading' of type 'phong':
 .PP
 .RS
-\fB@appname@ -b shading:duration=5.0:shading=phong\fR
+\fB@appname@ \-b shading:duration=5.0:shading=phong\fR
 .RE
 .PP
 To run a series of benchmarks use the \fB\-b\fR, \fB\-\-benchmark\fR command
 line option multiple times:
 .PP
 .RS
-\fB@appname@ -b shading:duration=5.0 -b build:use-vbo=false -b texture\fR
+\fB@appname@ \-b shading:duration=5.0 \-b build:use-vbo=false \-b texture\fR
 .RE
 .PP
 To set default option values for benchmarks:
 .PP
 .RS
-\fB@appname@ -b :duration=2.0 -b shading -b build -b :duration=5.0 -b texture\fR
+\fB@appname@ \-b :duration=2.0 \-b shading \-b build \-b :duration=5.0 \-b texture\fR
 .RE
 .PP
 
